Src/node_builder.py

```
@@ -207,17 +181,13 @@ def build_input(self, parent: str | int) -> str | int:
-            logic = InputLayerNode.create_input,
-            annotations = {
-                    "shape": Parameter(AttrType.INPUT, ANode[Single[object]]),
-                },
-            input=False,
-            output=LayerNode
-            )
+        layer = None
+        for group in self.node_list.values():
+            for group2 in group.values():
+                for node in group2:
+                    if node.node_type is not InputLayerNode: continue
```

Ruff (0.15.21)

[error] Multiple statements on one line (colon)

(E701)